packages/persist/lib/records.ts

```diff
@@ -32,10 +31,9 @@ export async function persistRecords({
     persistType: PersistType;
     accountId: number;
     environmentId: number;
-    connectionId: string;
+    connectionId: number;
```

Comment:
the external connectionId was only used for internal error reporting. Removing it to prevent confusion